src/components/App.js

```diff
@@ -31,22 +30,27 @@ class App extends React.Component {
     const { total, next } = this.state;
 
     return (
-      <section className="section1">
-        <p>Let&apos;s do some math!</p>
-        <div id="calcContainer">
-          <div id="cBody">
-            <div className="calcTop">
-              <Display result={
-                (total === null && next === null) ? '0' : next === null ? total : next
-                }
-              />
-            </div>
-            <div className="calcBtm">
-              <ButtonPanel onClick={this.handleClick} />
-            </div>
-          </div>
-        </div>
-      </section>
+      <CalculatorPage
+        total={total}
+        next={next}
+        handleClick={this.handleClick}
+      />
+      // <section className="section1">
+      //   <p>Let&apos;s do some math!</p>
+      //   <div id="calcContainer">
+      //     <div id="cBody">
+      //       <div className="calcTop">
+      //         <Display result={
+      //           (total === null && next === null) ? '0' : next === null ? total : next
+      //           }
+      //         />
+      //       </div>
+      //       <div className="calcBtm">
+      //         <ButtonPanel onClick={this.handleClick} />
+      //       </div>
+      //     </div>
+      //   </div>
+      // </section>
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**_Kindly remove before merging._**